src/otx/core/model/detection.py

```diff
@@ -443,7 +443,7 @@ def _customize_outputs(
                 tv_tensors.BoundingBoxes(
                     output.pred_instances.bboxes,
                     format="XYXY",
-                    canvas_size=output.img_shape,
+                    canvas_size=output.ori_shape,
```

Comment:
Why is this changed? `img_shape` is the final image shape which lies in front of model's `forward()`. On the other hand `ori_shape` is the image shape where we can get by decoding it from the file.

Reply:
`img_shape` is the input image size and not the original image size. Predicted boxes are rescaled to the original image size, so sizes don't match, therefore you will get boxes not within canvas size

Reply:
Do you mean that `output.pred_instances.bboxes` is rescaled to the original input image size?

Reply:
Yes, `output.pred_instances.bboxes` is in the same scaled as the original image size `ori_shape`. Rescaling happened in `mmdet.predict`